File: src/Shared/Domain/Entities/SightReview.cs
namespace Domain.Entities;
public class SightReview : BaseAuditableEntity
{
    public Sight Sight { get; set; }
    public Guid SightId { get; set; }

    public ICollection<SightReviewPhoto>? Photos { get; set; }

    public DateTime Posted { get; set; }
    public Guid OwnerId { get; set; }

    public string? Reviewer { get; set; }
    public double Rating { get; set; }
    public string? ReviewText { get; set; }
}

File: src/ClientApp/src/app/core/Models/SightReview.ts
import {Guid} from "guid-typescript";
import {SightReviewPhoto} from "./SightReviewPhoto";

export class SightReview {
  constructor(public id: Guid,
              public rating: number,
              public posted: Date,
              public sightId: Guid,
              public reviewText?: string,
              public reviewer?: string,
              public photos?: SightReviewPhoto[]) {
  }
}
